rabbitmq/rabbitmq.conf

```diff
@@ -0,0 +1,23 @@
+loopback_users = none
+listeners.tcp.default = 5672
+default_pass = guest
+default_user = guest
+hipe_compile = false
+management.listener.port = 15672
+management.listener.ssl = false
+stomp.default_user = guest
+stomp.default_pass = guest
+stomp.listeners.tcp.1 = 61613
```

Comment:
I don't think this works now cause the docker port is still 15674?

Reply:
Should change this to 15674, seeing that everything else is using that port

Reply:
removed conf

Reply:
> I don't think this works now cause the docker port is still 15674?

kinda realised I understood wrong, just gonna point out here for other ppl that this port is supposed to be 61613 coz this is for stomp, and 15674 is for webstomp